great-idea/css/index.css

```
@@ -63,4 +63,206 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+/* NAVBAR */
+
+.navigation {
+	width: 100%;
+	background-color: rgb(129, 223, 160); 
+	position: fixed;
+	z-index: 5;
+	display: flex;
+	top: 0;
+	justify-content: space-evenly;
+	justify-content: baseline;
+	margin-bottom: 80px;
+	flex-wrap: wrap;
+}
+
+nav {
+	display: flex;
+	align-items: center;
+	flex-wrap: wrap;
+}
+
+a {
+	text-decoration: none;
+	color: black;
+	font-size: 15px;
+}
+
+nav a {
+	padding-right: 10px;
+	padding-left: 10px;
+	margin: 0 30px;
+	display: flex;
+	align-items: center;
+	height: 35px;
+}
+
+nav a:hover {
+	animation-name: colorchange;
+	animation-duration: .3s;
+	animation-fill-mode: forwards;
+	box-shadow: -2px 2px rgba(0, 0, 0, 0.541);
+}
+@keyframes colorchange {
+	from {background:#81DFA0;}
+	to {background:#fae;}
+}
+@keyframes colorchange2 {
+	from {background:#81DFA0;}
+	to {background: white;}
+}
+
+h1 {
+
+	padding-right: 20px;
+	padding-left: 30px;
+	margin-top: 8px;
+	height: 40px;
+	padding-top: 2px;
+}
+
+h1:hover {
+	animation-name: colorchange2;
+	animation-duration: .3s;
+	animation-fill-mode: forwards;
+	box-shadow: -2px 2px rgba(0, 0, 0, 0.541);
+}
+
+.spacer {
+	margin-bottom: 100px;
+}
+
+/* HEADER */
+header {
+	/* margin: 50px auto 50px auto;
+	width: 1000px; */
+	z-index: 0;
+	display: flex;
+	justify-content: space-evenly;
+	align-items: center;
+	flex-wrap: wrap;
+}
```

```suggestion
	margin-bottom: 70px;
```
Content needs to be separated a bit, adding some margin here will help with that.